app/views/users/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#index</h1>
+<p>Find me in app/views/users/index.html.erb</p>
```

Comment:
- [ ] There should be corresponding view files for each of the controller actions. Currently, the show view file is missing for the user. [Hint: `app/views/users/show.html.erb `]

Reply:
Done ✔